dapper/dapperlib/types.go

```diff
@@ -82,7 +82,7 @@ var daFuncs = map[DataAggrMethod]DataAggrFunc{
 		if len(in) == 0 {
 			return ""
 		}
-		var tot = math.MaxFloat64
+		var tot = 0.0
```

Comment:
This seems like a very specific mistake, not sure if there's a reason why a total would start at max float - I can't think of anything...